src/library/scala/collection/Parallelizable.scala

```diff
@@ -43,6 +46,31 @@ trait Parallelizable[+A, +ParRepr <: Parallel] extends Any {
     cb.result()
   }
 
+  /** Returns a parallel implementation of this collection using the given thread pool.
+   *
+   * @param pool the thread pool to use for supporting parallelism
+   * @return a parallel implementation of this collection using `pool` for its threading
+   */
+  final def parWith(pool: ForkJoinPool): ParRepr = parWith(new ForkJoinTaskSupport(pool))
+
+  /** Returns a parallel implementation of this collection using the given task support.
+   *
+   * @param ts the task support to use for supporting parallelism
+   * @return a parallel implementation of this collection using `ts` for parallelism
+   */
+  final def parWith(ts: TaskSupport): ParRepr = {
```

Comment:
I'd prefer to have param name `taskSupport` here so that people can leverage named parameters if they feel their code is not clear enough without it.

(Might be a good idea for the first overload, too? `pool` -> `threadPool`)